.github/actions/assimp-install-dep/action.yml

```diff
@@ -41,6 +41,7 @@ runs:
         cmake ../assimp
         -DASSIMP_BUILD_ALL_IMPORTERS_BY_DEFAULT=OFF
         -DASSIMP_BUILD_ASSIMP_TOOLS=OFF
+        -DASSIMP_BUILD_3MF_IMPORTER=ON
```

Comment:
we need to think about adding that to the superbuild as well

Reply:
for X too

Reply:
https://github.com/f3d-app/f3d-superbuild/pull/141